### PR TITLE
[Teams] auth domain > team domain

### PR DESCRIPTION
**Pull request by abracchi-tw**

### CODE REVIEW

**Review by TownLake — APPROVED** [review]

File: products/cloudflare-one/src/content/connections/connect-devices/warp/deployment/Windows-Teams.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ If your organization uses Teams Access policies to control device registration,
 1. Next, click **Preferences**.
 1. Select the **Account** tab.
 1. Click **Login with Cloudflare for Teams**.
-1. Enter your organization name (if your auth domain were `https://example.cloudflareaccess.com`, you would enter `example`).
+1. Enter your [team name](/glossary#team-name).
```
Comment (kennyj42) [inline products/cloudflare-one/src/content/connections/connect-devices/warp/deployment/Windows-Teams.md:87]: Are we doing these changes in the docs before the UI changes or just queuing up?

File: products/cloudflare-one/src/content/identity/idp-integration/azuread.md [hunk 2]
```diff
@@ -66,29 +71,37 @@ Copy the `Value` field of the client secret. Treat this value like a password. T
...
 
-AzureAD exposes directory groups in a format that consists of random strings, the `Object Id`, that is distinct from the `Name`. In the example below, the group named "Admins" has an ID of "61503835-b6fe-4630-af88-de551dd59a2".
+AzureAD exposes directory groups in a format that consists of random strings, the `Object Id`, that is distinct from the `Name`. In the example below, the group named "Admins" has an ID of `61503835-b6fe-4630-af88-de551dd59a2`.
 
 ![Azure AD Test Connection](../../static/documentation/identity/azure/object-id.png)
 
-When configuring Access to use Azure groups, you must input the `Object Id`.
+To configure Access to use Azure groups, make sure you toggle on the **Support groups** switch.
+
+![Azure AD groups](../../static/documentation/identity/azure/azure-ad-groups.png)
```
Comment (kennyj42) [inline products/cloudflare-one/src/content/identity/idp-integration/azuread.md:102]: this image is broken

File: products/cloudflare-one/src/content/identity/idp-integration/facebook-login.md [hunk 3]
```diff
@@ -15,48 +15,49 @@ Use these steps to set up Facebook as your identity provider.
...
+12. Click **Settings > Basic** on the left-hand menu.
 
     ![Facebook Settings](../../static/documentation/identity/facebook/fb6.png)
 
-14. In the **Cloudflare** **Access** app, copy the App ID and App Secret into the **OAuth ID** and **OAuth Secret** fields.
+13. On the Teams dashboard, navigate to **Access > Authentication**.
+14. Click *+ Add* under **Login Methods**, and select Facebook as your IdP. 15. Copy the App ID and App Secret into the **OAuth ID** and **OAuth Secret** fields.
 
-    ![OAuth ID and OAuth Secret](../../static/documentation/identity/facebook/fb7.png)
+    ![OAuth ID and OAuth Secret](../../static/documentation/identity/facebook/fb10.png)
```
Comment (kennyj42) [inline products/cloudflare-one/src/content/identity/idp-integration/facebook-login.md:51]: This image looks broken

File: products/cloudflare-one/src/content/identity/idp-integration/keycloak.md [hunk 4]
```diff
@@ -8,43 +8,40 @@ Keycloak is an open source identity and access management solution built by JBos
...
-    IdP Entity ID or Issuer URL: `https://<unique_id>.cloudflareaccess.com/cdn-cgi/access/callback`
-    Signing certificate: Use the X509 Certificate in the Realm Settings from Keycloak
+    | Field | Example |
+    | ----- | ------- |
+    | Single Sign-On URL | `https://<keycloak_domain>/auth/realms/master/protocol/saml` |
+    | IdP Entity ID or Issuer URL | `https://<unique_id>.cloudflareaccess.com/cdn-cgi/access/callback` |
+    | Signing certificate | Use the X509 Certificate in the Realm Settings from Keycloak |
 
-    ![Access Config](../../static/documentation/identity/keycloak/access-config.png)
+    ![Access Config](../../static/documentation/identity/keycloak/access-config-keycloak.png)
```
Comment (kennyj42) [inline products/cloudflare-one/src/content/identity/idp-integration/keycloak.md:49]: broken image

Comment (kennyj42) [reply products/cloudflare-one/src/content/identity/idp-integration/keycloak.md:49]: \it's the old dash I think

File: products/cloudflare-one/src/content/identity/idp-integration/okta.md [hunk 5]
```diff
@@ -8,66 +8,57 @@ You can integrate Okta with Cloudflare Access to allow users to reach applicatio
 
 1. In your Okta dashboard, click **Admin**.
```
Comment (kennyj42) [inline products/cloudflare-one/src/content/identity/idp-integration/okta.md:9]: The numbering is broken on this one. It's all "1." 's

File: products/cloudflare-one/src/content/identity/idp-integration/saml-okta.md [hunk 6]
```diff
@@ -38,9 +38,7 @@ To set up SAML with Okta as your IdP:
 
     ![Okta SAML Settings card](../../static/documentation/identity/saml-okta/saml-okta-5.png)
 
-1. In the **Single sign on URL** and the **Audience URI** **(SP Entity ID)** fields, enter your authorization domain, and include this  callback at the end of the path: `/cdn-cgi/access/callback`.
-
-    **Tip**: You can find your organization’s authorization domain in Cloudflare Access. It begins with a subdomain unique to your organization and ends with the domain `cloudflareaccess.com`, including the callback path specified above.
+1. In the **Single sign on URL** and the **Audience URI** **(SP Entity ID)** fields, enter your [team domain](/glossary#team-domain) followed by this callback at the end of the path: `/cdn-cgi/access/callback`.
```
Comment (kennyj42) [inline products/cloudflare-one/src/content/identity/idp-integration/saml-okta.md]: I think this should have an e.g.:

e.g. [your-team-name].cloudflareaccess.com/cdn-cgi/access/callback

File: products/cloudflare-one/src/content/identity/idp-integration/yandex.md [hunk 7]
```diff
@@ -23,13 +23,7 @@ To set up Yandex for Cloudflare Access:
...
 
 1. Select **Platform** and click **Web Services.**
-1. In the **Callback URL #1** field, input your authorization domain.
-
-    Replace “your-domain” with the authentication domain listed in Cloudflare Access, and include the callback in the path:
-
-    ```txt
-    https://your-domain.cloudflareaccess.com/cdn-cgi/access/callback
-    ```
+1. In the **Callback URL #1** field, enter [team domain](/glossary#team-domain) followed by this callback at the end of the path: `/cdn-cgi/access/callback`.
```
Comment (kennyj42) [inline products/cloudflare-one/src/content/identity/idp-integration/yandex.md]: Same comment as signed_authn

Comment (kennyj42) [reply products/cloudflare-one/src/content/identity/idp-integration/yandex.md]: should give the e.g.

File: products/cloudflare-one/src/content/identity/idp-integration/yandex.md [hunk 8]
```diff
@@ -54,21 +48,10 @@ To set up Yandex for Cloudflare Access:
...
-1. In the Callback URI #1 field, input your authorization domain.
-
-    ```txt
-    https://your-domain.cloudflareaccess.com/cdn-cgi/access/callback
-    ```
-
-    Replace “your-domain” with the authentication domain listed in Cloudflare Access, and include the callback in the path.
-
-1. Click **Save and test**.
+     ![Cloudflare Access Add a Yandex identity provider dialog Platforms](../../static/documentation/identity/yandex/yandex-9.png)
```
Comment (kennyj42) [inline products/cloudflare-one/src/content/identity/idp-integration/yandex.md]: this image may be broken

**Review by kennyj42 — CHANGES_REQUESTED** [review]
See individual comments. A couple image issues and a few spots where additional examples may help

File: products/cloudflare-one/src/content/connections/connect-devices/warp/deployment/Windows-Teams.md (hunk 1 above)
Comment (abracchi-tw) [reply products/cloudflare-one/src/content/connections/connect-devices/warp/deployment/Windows-Teams.md:87]: yeah these changes will go live before the UI gets updated, but it's just a matter of days

File: products/cloudflare-one/src/content/identity/idp-integration/okta.md (hunk 5 above)
Comment (abracchi-tw) [reply products/cloudflare-one/src/content/identity/idp-integration/okta.md:9]: yeah that's intended, they'll be rendered correctly

File: products/cloudflare-one/src/content/identity/idp-integration/saml-okta.md (hunk 6 above)
Comment (abracchi-tw) [reply products/cloudflare-one/src/content/identity/idp-integration/saml-okta.md]: fixed

File: products/cloudflare-one/src/content/identity/idp-integration/yandex.md (hunk 7 above)
Comment (abracchi-tw) [reply products/cloudflare-one/src/content/identity/idp-integration/yandex.md]: fixed

**Review by kennyj42 — APPROVED** [review]

**Pull request merged**